packages/commentor/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=commentor
+pkgver=18.548094d
+pkgrel=1
+pkgdesc='Extract all comments from the specified URL resource.'
+groups=('blackarch' 'blackarch-code-audit')
+arch=('any')
+url='https://github.com/D3vil0p3r/comMENTOR'
+license=('GPL3')
+depends=('bash')
+makedepends=('git')
+source=("git+https://github.com/D3vil0p3r/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+  mkdir -p "$pkgdir/usr/share/commentor"
+  mkdir -p "$pkgdir/usr/bin"
+  install -Dm 755 -t "$pkgdir/usr/share/commentor/" commentor
```

Comment:
You could deploy multiple files in one line, cf our templates at https://github.com/BlackArch/blackarch-pkgbuilds
also

Reply:
Thank you @noraj for your tip. Solved on commit dd7124536ed647aee42aa7d99cad4bb919665036